SocialCareCaseViewerApi/V1/UseCase/FormSubmissionsUseCase.cs

```diff
@@ -168,14 +170,25 @@ private static void UpdateSubmissionState(CaseSubmission submission, UpdateCaseS
                     {
                         throw new UpdateSubmissionException($"Invalid submission state change from {mapSubmissionStateToResponseString[submission.SubmissionState]} to {mapSubmissionStateToResponseString[newSubmissionState]}");
                     }
-                    if (submission.Workers.Any(submissionWorker => submissionWorker.Email.ToLower().Equals(request.EditedBy.ToLower())))
+                    if (submission.Workers.Any(submissionWorker => submissionWorker.Email.ToUpperInvariant().Equals(request.EditedBy.ToUpperInvariant(), StringComparison.InvariantCulture)))
                     {
                         throw new UpdateSubmissionException($"Worker with email {request.EditedBy} cannot approve the submission as they have worked on the submission");
                     }
                     submission.ApprovedAt = DateTime.Now;
                     submission.ApprovedBy = worker;
                     break;
-
+                case SubmissionState.PanelApproved:
+                    if (submission.SubmissionState != SubmissionState.Approved)
+                    {
+                        throw new UpdateSubmissionException($"Invalid submission state change from {mapSubmissionStateToResponseString[submission.SubmissionState]} to {mapSubmissionStateToResponseString[newSubmissionState]}");
+                    }
+                    if (submission.Workers.Any(submissionWorker => submissionWorker.Email.ToUpperInvariant().Equals(request.EditedBy.ToUpperInvariant(), StringComparison.InvariantCulture)))
+                    {
+                        throw new UpdateSubmissionException($"Worker with email {request.EditedBy} cannot approve the submission as they have worked on the submission");
```

Comment:
are we testing whether someone _worked_ on it or just _created/started_ it? the intention was just to check if it was created by them

Reply:
I always thought worked on made more sense, if a social worker started a form and a second social worker also added some info. Should the second worker be allowed to approve the work? I'm happy to change it if you think so? Does the frontend only care about createdBy, it would obviously not be ideal to show the UI for something the backend does not allow?